docs/api/I13nUtils.md

```diff
@@ -0,0 +1,40 @@
+## I13nUtils
+
+The `I13Utils` mixin provides util functions for you to access the i13n nodes and execute i13n functionalities
```

Comment:
Thats fine if its not needed. It was mainly for components that didn't want to use createI13nNode right?